tests/e2e/e2e_common.go

```diff
@@ -55,6 +55,9 @@ const (
 	envRegionZoneWithNoSharedDS                = "TOPOLOGY_WITH_NO_SHARED_DATASTORE"
 	envRegionZoneWithSharedDS                  = "TOPOLOGY_WITH_SHARED_DATASTORE"
 	envSharedDatastoreURL                      = "SHARED_VSPHERE_DATASTORE_URL"
+	envSharedVVOLDatastoreURL                  = "SHARED_VVOL_DATASTORE_URL"
+	envSharedNFSDatastoreURL                   = "SHARED_NFS_DATASTORE_URL"
+	envSharedVMFSDatastoreURL                  = "SHARED_VMFS_DATASTORE_URL"
```

Comment:
Don't we have any other way to determine datastore URL ?

Reply:
In our jenkin jobs we are determining the datastore URL's through govc commands and then setting it to these environment variables before running the E2E tests. 

As of now in the go code we have not made those changes to auto read the URL's. We will take that up once we have some bandwidth.

Reply:
In that case, can you add a `//TODO:` or create an issue to track the same. Otherwise, the ENVs will keep growing